### PR TITLE
Update fullnode.md

### PR DESCRIPTION
**Pull request by grossbel12**
Ad chapter Typical issues. This section will show common problems and solutions for people who are not familiar with installing nodes and who are setting up a node for the first time. For instance, I add typical error 98/99.

### CODE REVIEW

File: doc/src/build/fullnode.md [hunk 1]
```diff
@@ -273,3 +273,10 @@ state of the chain from genesis.
 
 Please see our [privacy policy](https://sui.io/policy/) to learn how we handle
 information about our nodes.
+
+## Typical isues
```
Comment (randall-Mysten) [inline doc/src/build/fullnode.md:277]:
```suggestion
## Resolving errors 
```

File: doc/src/build/fullnode.md [hunk 2]
```diff
@@ -273,3 +273,10 @@ state of the chain from genesis.
 
 Please see our [privacy policy](https://sui.io/policy/) to learn how we handle
 information about our nodes.
+
+## Typical isues 
+1. Error 98/99  ![Error 98](https://user-images.githubusercontent.com/86624298/183159157-46971a06-a5c4-4196-80b0-63177a78164d.png)
```
Comment (randall-Mysten) [inline doc/src/build/fullnode.md:278]:
```suggestion
#### Error 98/99
```

File: doc/src/build/fullnode.md [hunk 3]
```diff
@@ -273,3 +273,10 @@ state of the chain from genesis.
 
 Please see our [privacy policy](https://sui.io/policy/) to learn how we handle
 information about our nodes.
+
+## Typical isues 
+1. Error 98/99  ![Error 98](https://user-images.githubusercontent.com/86624298/183159157-46971a06-a5c4-4196-80b0-63177a78164d.png) 
+You should restart your server ```shell $ shutdown -r now  ``` and after that will run Restart your Sui fullnode  ```shell
+    $ cargo run --release --bin sui-node -- --config-path fullnode.yaml
+    ```
```
Comment (randall-Mysten) [inline doc/src/build/fullnode.md:281]:
```suggestion
If you receive an error with code 98 or 99, you should restart your server using the following command:
`$ shutdown -r now` 

Then, restart your Sui fullnode:
`$ cargo run --release --bin sui-node -- --config-path fullnode.yaml`
```

Is there an extra -- in the command?
--bin sui-node -- --config-path

**Review by randall-Mysten — COMMENTED** [review]
Thanks for submitting this contribution. A couple of questions for you.

**Comment by randall-Mysten** [comment]
Hey folks, I am working with grossbel12 after some chatting in Discord. This is a trial run with a new user, thanks for your patience. We'll fix the branch etc.

**Review by huitseeker — CHANGES_REQUESTED** [review]
I think the PR could be better if it indicated 
1. The 98 error code is only valid in Linux, in windows it's a code 10048. 
2. The reason for the error is that one of the bind addresses used in the configuration is already in use on the system. 
3. Finding what other thing is using the port and managing who gets what resource is key to resolving this. There are many articles on the internet on how to dig into this, [e.g.](https://handsonnetworkprogramming.com/articles/bind-error-98-eaddrinuse-10048-wsaeaddrinuse-address-already-in-use/).
4. restarting the server is not the only nor necessarily the right solution. Rather the documentation should indicate: "as one of the ways to address this, you could restart your system, shutting down servers that might have been started on the system".

**Comment by Clay-Mysten** [comment]
Thanks for the contribution @grossbel12 , coordination @randall-Mysten, and input @huitseeker. Randall, we can discuss how to improve this in our meeting today.

It would be good to have more than one issue. I wonder if @tharbert, @bmwill , @mystenmark , @allan-bailey or @ebmifa have other known fullnode issues we can include here too. Thanks, all.

**Comment by Clay-Mysten** [comment]
Asking @bmwill to take a look.

**Comment by grossbel12** [comment]
I created new common error in my repositories, I send message to @randall-Mysten. If the topic with common problems with Fullnode is relevant, I'll add some.

**Comment by Clay-Mysten** [comment]
> I created new common error in my repositorie, I send message to @randall-Mysten. If the topic with common problems with Fullnode is relevant, I'll add some.

Please include the common errors here. Thanks!

**Comment by grossbel12** [comment]
> > I created new common error in my repositorie, I send message to @randall-Mysten. If the topic with common problems with Fullnode is relevant, I'll add some.
> 
> Please include the common errors here. Thanks!

Done check plz

**Comment by randall-Mysten** [comment]
> > > I created new common error in my repositorie, I send message to @randall-Mysten. If the topic with common problems with Fullnode is relevant, I'll add some.
> > 
> > 
> > Please include the common errors here. Thanks!
> 
> Done check plz

Thank you. I see some comments from huitseeker that are still pending. Do you want to try and address those?

**Comment by Clay-Mysten** [comment]
> > > > I created new common error in my repositorie, I send message to @randall-Mysten. If the topic with common problems with Fullnode is relevant, I'll add some.
> > > 
> > > 
> > > Please include the common errors here. Thanks!
> > 
> > 
> > Done check plz
> 
> Thank you. I see some comments from huitseeker that are still pending. Do you want to try and address those?

Actually, this entire solution is covered in:
https://docs.sui.io/devnet/build/fullnode#built-from-source

So I agree with @huitseeker we should qualify the error with your environment as the new information here. Can you confirm your operating system and version? Also, please send a higher resolution version of the image as it is blurry at full size:
https://user-images.githubusercontent.com/86624298/184237312-4af31e15-c939-47e9-8055-535bee579dba.png

Thanks, @grossbel12 !

**Comment by randall-Mysten** [comment]
I am closing this PR as it is stale and some of the issues are being addressed with other content updates. Please submit another PR if there are still unresolved issues. Thanks for contributing to Sui.

**Pull request closed without merging**